atomdb/__init__.py

```diff
@@ -13,43 +13,75 @@
 # You should have received a copy of the GNU General Public License
 # along with AtomDB. If not, see <http://www.gnu.org/licenses/>.
 
-r"""AtomDB, a database of atomic and ionic properties."""
+"""AtomDB console script.
 
-import importlib
-from importlib.metadata import PackageNotFoundError
+This script provides a command-line interface to compile or query entries in the AtomDB database.
+"""
 
-from atomdb.periodic import Element
+import sys
+from argparse import ArgumentParser, ArgumentTypeError
+from atomdb import compile_species, load
 
-from atomdb.species import Species
 
-from atomdb.promolecule import Promolecule
+def positive_int(value):
+    """Validate if the argument is a positive integer."""
+    try:
+        ivalue = int(value)
+        if ivalue <= 0:
+            raise ArgumentTypeError(f"{value} is not a positive integer.")
+        return ivalue
+    except ValueError as e:
+        raise ArgumentTypeError(f"Invalid integer value: {value}") from e
 
-from atomdb.periodic import element_number, element_symbol, element_name
 
-from atomdb.species import compile_species, load, dump, raw_datafile
+def main():
```

Comment:
Having a main function to organize the CLI code is a good idea. However, this refactor has to be made in the `__main__.py` module rather than here.
Also, given that there is another CLI implementation in [`__main__.py`](https://github.com/theochem/AtomDB/blob/464f116f030da1ad05c28ae028e25985ecbed2ea/atomdb/__main__.py#L16), do you know which code is getting executed (the one in `__main__.py` or the one in here)?

you can take a look at the use of the `__main__` file in the Python documentation [here](https://docs.python.org/3/library/__main__.html#main-py-in-python-packages)